pkg/translator/opencensus/oc_to_resource.go

```diff
@@ -127,9 +126,9 @@ func ocNodeResourceToInternal(ocNode *occommon.Node, ocResource *ocresource.Reso
 		for k, v := range ocResource.Labels {
```

Comment:
Shouldn't filing it from resource labels go first now to maintain the same precedence in case of conflict with node labels?

Reply:
Sure, don't think that matters in practice since they should not send that info as resource.